### ISSUE TITLE
name 'glyph' is not defined

### ISSUE THREAD

**Issue by rettinghaus** [fkretlow/fontforge-smufl#3]
In the static method `canonical_glyphname` it tries to get `glyph.glyphname`, which obviously cannot work, as it is not defined here.

**Comment by fkretlow** [comment]
Good catch, thanks. It's quite funny that this hasn't been spotted earlier. I'll try to fix it when I get a chance.
